src/main/jbake/content/documentation/bundles/scripting.md

```diff
@@ -67,9 +67,143 @@ Mapping `html` to Thymeleaf 3.0:
+
+  1. `sling.servlet.resourceTypes:List` - mandatory; defines the provided resource type; its value is a list of resource types
+  2. `sling.servlet.selectors:List` - optional; defines the list of selectors that this resource type can handle;
+  3. `sling.servlet.extensions:List` - optional; defines the list of extensions that this resource type can handle;
+  4. `sling.servlet.methods:List` - optional; defines the list of HTTP methods that this resource type can handle;
+  5. `version:Version` - optional; defines the version of the provided `resourceType`;
+  6. `extends:String` - optional; defines which resource type it extends; the version range of the extended resource type is defined in a
+    `Require-Capability`.
+
+The `BundledScriptTracker` will register a Sling Servlet with the appropriate properties for each `sling.servlet` capability. The
+servlets will be registered using the bundle context of the bundle providing the `sling.servlet` capability, making
+sure to expose the different versions of a resource type as part of the registered servlet's properties. On top of this, a plain resource
+type bound servlet will also be registered, which will be automatically wired to the highest version of the `resourceType`. All the
+mentioned service registrations are managed automatically by the `BundledScriptTracker`.
+
+### So how do I deploy my scripts?
+Short answer: exactly like you deploy your code, preferably right next to it. Pack your scripts using the following conventions:
+
+  1. create a `src/main/resources/javax.script` folder in your bundle (if you want to embed the scripts as they are) or just put the
+   scripts in `src/main/scripts` if you want to precompiled them (e.g. JSP and HTL);
+  2. each folder under the above folders will identify a `resourceType`;
+  3. inside each `resourceType` folder you can optionally create a `Version` folder; this has to follow the Semantic Versioning
+   constraints described at [1];
+  4. add your scripts, using the same [naming conventions that you were used to from before][5];
+  5. manually define your provide and require capabilities; just kidding; add the
+  [`scriptingbundle-maven-plugin`](https://github.com/apache/sling-scriptingbundle-maven-plugin) to your build section and add its required
+  properties in the `maven-bundle-plugin`'s instructions (check [these examples](https://github.com/apache/sling-org-apache-sling-scripting-bundle-tracker-it/tree/master/examples/));
+  6. `mvn clean sling:install`.
+
+### Integration Tests
+
+The integration tests for bundled scripts are provided by the [`org.apache.sling.scripting.bundle.tracker.it`](https://github.com/apache/sling-org-apache-sling-scripting-bundle-tracker-it) project.
+
+## Script resolution order
+
+The [same rules as for servlets][6] are being followed but in addition keep in mind that bundled scripts (as well as servlets) are prefered over resource scripts.
```

Comment:
@karlpauls Is this observation correct? I couldn't find according tests for that....